group_vars/mysql/public.yml

```diff
@@ -0,0 +1,30 @@
+MYSQL_SERVER_DOMAIN: "{{ inventory_hostname }}"
+
+MYSQL_SERVER_BACKUP_DIR: '/var/lib/mysql/backup'
+TARSNAP_BACKUP_PRE_SCRIPT: "/usr/local/sbin/backup-pre.sh"
+TARSNAP_BACKUP_POST_SCRIPT: "/usr/local/sbin/backup-post.sh"
+TARSNAP_BACKUP_FOLDERS: "{{ MYSQL_SERVER_BACKUP_DIR }}"
+TARSNAP_ARCHIVE_NAME: "{{ MYSQL_SERVER_DOMAIN }}"
+
+mysql_root_password: "{{ MYSQL_ROOT_PASSWORD }}"
+mysql_root_password_update: true
+
+mysql_packages:
+  - mysql-server-5.5
+  - mysql-client-5.5
+
+FORWARD_MAIL_MYDOMAIN: "{{ MYSQL_SERVER_DOMAIN }}"
+UNATTENED_UPGRADES_ERRORS_RELAY_TO: "{{ MYSQL_SERVER_DOMAIN }}"
+
+SANITY_CHECK_DISK_SPACE:
+  - mount: /
+    treshold_mb: 5000
+  - mount: /var/lib/mysql
+    treshold_mb: 5000
```

Comment:
Damn :( See [this one](https://github.com/open-craft/ansible-sanity-checker/pull/1/files/8c7a536bc88867fd75f7081b7a806ae811a86fd3#r60913763). I forgot to update variables here (which means that this role is using default of 20%)